drivers/dma/dma_mcux_lpc.c

```diff
@@ -374,6 +374,8 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 		data = DEV_CHANNEL_DATA(dev, virtual_channel);
 	}
 
+	memset(data, 0, sizeof(struct channel_data));
```

Comment:
Do we need a similar initialization in the dma_mcux_edma.c: dma_mcux_edma_configure() ?